client-js/app/elements/labrad-registry.html

```diff
@@ -97,6 +98,10 @@
     dir-item {
       white-space: nowrap;
     }
+    .mono {
+      font-family: 'roboto mono', monospace;
```

Comment:
Yup, it works (icognito, offline on the right):
![screenshot from 2016-01-25 14 54 02](https://cloud.githubusercontent.com/assets/8173925/12567390/ab4d1d62-c373-11e5-9575-2b425980f547.png)